test/Properties.hs

```diff
@@ -43,11 +46,23 @@ t_normalize mode = t_nonEmpty $ T.normalize mode
 #endif
 
 main :: IO ()
-main = do
+main =
+      hspec
+    $ H.parallel
+    $ modifyMaxSuccess (const 10000)
+    $ do
+        prop "quotRem28" $ \(n :: Word) ->
+            (fromIntegral n) `quotRem` 28 == quotRem28 (fromIntegral n)
+        prop "quotRem28 maxBound" $ \(n :: Word) ->
+            let n1 = maxBound `div` 2 - n
+             in (fromIntegral n1) `quotRem` 28 == quotRem28 (fromIntegral n1)
+        prop "quotRem21" $ \(n :: Word) ->
+            (fromIntegral n) `quotRem` 21 == quotRem21 (fromIntegral n)
+        prop "quotRem21 maxBound" $ \(n :: Word) ->
+            let n1 = maxBound `div` 2 - n
+             in (fromIntegral n1) `quotRem` 21 == quotRem21 (fromIntegral n1)
```

Comment:
I think it could look simpler with `NonNegative` wrapper:

```haskell

import Test.QuickCheck (NonNegative(..))
...
        prop "quotRem28" $ \(NonNegative n) -> n `quotRem` 28 == quotRem28 n
        prop "quotRem28 maxBound" $ \(NonNegative n') ->
            let n = maxBound - n' in n `quotRem` 28 == quotRem28 n
        prop "quotRem21" $ \(NonNegative n) -> n `quotRem` 21 == quotRem21 n
        prop "quotRem21 maxBound" $ \(NonNegative n') ->
            let n = maxBound - n' in n `quotRem` 21 == quotRem21 n
```